.github/workflows/docker-multi-arch.yml

```
@@ -33,70 +94,398 @@ jobs:
+        run: |
+          VERSION=${{ needs.calculate-version.outputs.image_version }}
+
+          docker manifest create \
+            ${{ secrets.DOCKERHUB_USERNAME }}/n.e.k.o:${VERSION}-full \
+            --amend ${{ secrets.DOCKERHUB_USERNAME }}/n.e.k.o:${VERSION}-full-linux-amd64 \
+            --amend ${{ secrets.DOCKERHUB_USERNAME }}/n.e.k.o:${VERSION}-full-linux-arm64
+          docker manifest push ${{ secrets.DOCKERHUB_USERNAME }}/n.e.k.o:${VERSION}-full
+
+      - name: Create and push CI short tags (GHCR)
+        if: needs.calculate-version.outputs.is_main_commit == 'true'
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:11:41: Double quote to prevent globbing and word splitting

(shellcheck)